Firestore/Source/Local/FSTMemoryRemoteDocumentCache.mm

```diff
@@ -18,23 +18,23 @@
 
 #import "Firestore/Source/Core/FSTQuery.h"
 #import "Firestore/Source/Model/FSTDocument.h"
-#import "Firestore/Source/Model/FSTDocumentDictionary.h"
 #import "Firestore/Source/Model/FSTDocumentKey.h"
 
-NS_ASSUME_NONNULL_BEGIN
-
-@interface FSTMemoryRemoteDocumentCache ()
+#include "Firestore/core/src/firebase/firestore/model/document_dictionary.h"
 
-/** Underlying cache of documents. */
-@property(nonatomic, strong) FSTMaybeDocumentDictionary *docs;
+using firebase::firestore::model::DocumentDictionary;
+using firebase::firestore::model::MaybeDocumentDictionary;
 
-@end
+NS_ASSUME_NONNULL_BEGIN
 
-@implementation FSTMemoryRemoteDocumentCache
+@implementation FSTMemoryRemoteDocumentCache {
+  /** Underlying cache of documents. */
+  MaybeDocumentDictionary _docs;
+}
 
 - (instancetype)init {
   if (self = [super init]) {
-    _docs = [FSTMaybeDocumentDictionary maybeDocumentDictionary];
+    _docs = MaybeDocumentDictionary{};
```

Comment:
Is this necessary? I presume `init` can only be called once during the lifetime of the object. Does Objective-C++ not call the default constructor otherwise?